backend/src/main/java/de/neuefische/backend/model/dto/IndexCardDto.java

```diff
@@ -0,0 +1,14 @@
+package de.neuefische.backend.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
```suggestion
@NoArgsConstructor
@Builder
```
Wenn du dich im Controller beim post mapping für ein Dto entscheidest, könntest du für die Tests hier ein Builder nutzen.